src/components/component.ts

```diff
@@ -527,6 +527,23 @@ export class Component {
   }
 
   /**
+   * @returns {Bounds} for the component in pixel space, where the topLeft
+   * represents the component's minimum x and y values and the bottomRight represents
+   * the component's maximum x and y values.
+   */
+  public bounds(): Bounds {
```

Comment:
should specify this is relative to its parent (since it's based off this.origin())